BuildResidentialHPXML/measure.rb

```diff
@@ -3151,7 +3206,8 @@ def self.set_water_heating_systems(hpxml, runner, args)
                                     recovery_efficiency: recovery_efficiency,
                                     related_hvac_idref: related_hvac_idref,
                                     standby_loss: standby_loss,
-                                    jacket_r_value: jacket_r_value)
+                                    jacket_r_value: jacket_r_value,
+                                    temperature: 125.0)
```

Comment:
And here.

Reply:
I decided to expose `water_heater_setpoint_temperature`.